src/styles/application.scss

```diff
@@ -0,0 +1,22 @@
+// This is the top-level scss file. Import all other files in here.
+
+/*
+While prototyping we load all of semantic-ui
+eventually we pair this down into calling only the subcomponents we want in our code base
+*/
+@import '~semantic-ui-sass/app/assets/stylesheets/semantic-ui/globals/all';
```

Comment:
This is how webpack's sass-loader likes to access packages in node modules, editing includePaths in the config (traditionally the sass way) was not working.